src/components/shared/alerts/alert.scss

```diff
@@ -0,0 +1,102 @@
+.notification_container {
+  font-size: 14px;
+  box-sizing: border-box;
+  position: fixed;
+  z-index: 1;
+
+  .notification {

```

Comment:
I thought you were gonna put this guy at the bottom left, rather than the top? Did you change your mind?

Reply:
you are correct i just forgot to carry the styles into the sheet from the browser